### PR TITLE
Nit: Motivation: nixos.org not confusing

### PR DESCRIPTION
**Pull request by rgoulter**
I'd argue that the domain name itself isn't "confusing". The word "incongruent" would be more apt.

Perhaps a simpler phrasing is that "the domain name nixos.org is not a close match for the website, which is about nix in general and not just about NixOS in particular".

### CODE REVIEW

**Comment by iFreilicht** [comment]
I would say "confusing" fits well. The domain-name alone has caused people I've sent it to to ask "Oh, so it's a Linux Distro?" or "but I don't use Linux", stuff like that.

**Comment by rgoulter** [comment]
Still, "incongruent" is more precise term.

**Comment by infinisil** [comment]
I had to look up (in)congruent in the dictionary, it does indeed match pretty well. I think we should just have both though

File: rfcs/1000-canonical-domain.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Use `nix.dev` as the canonical domain for all official Nix projects.
 # Motivation
 [motivation]: #motivation
 
-- `nixos.org` is confusing because it's not just about NixOS.
+- `nixos.org` is incongruent since it's not just about NixOS.
```
Comment (infinisil) [inline rfcs/1000-canonical-domain.md]: I do like your suggested wording though!

```suggestion
- The domain name nixos.org is not a close match for the website, which is about nix in general and not just about NixOS in particular
```

**Pull request merged**